### PR TITLE
fix(v1beta2): enable self signed jwt for grpc

### LINKED ISSUE #174: Enable self-signed JWT flow for v1beta2
Documentai uses a regional host (`us-documentai.googleapis.com`) as the default for v1beta2 so self-signed JWT cannot be used for v1beta2.

When the host is defined as `documentai.googleapis.com` re-enable the self-signed JWT flow.

### PR DESCRIPTION
**Pull request by nicain**
Remove Owlbot  v1beta2 customization that passes default scopes as user scopes, so that the auth library does not use the self-signed JWT flow.

Fixes #174

### CODE REVIEW

**Comment by nicain** [comment]
@busunkim96 Are there special Labels necessary to engage owlbot for this change?

**Comment by parthea** [comment]
@googlebot I consent.

File: owlbot.py [hunk 1]
```diff
@@ -36,15 +36,6 @@
...
-    s.replace(library / "google/cloud/documentai_v1beta2/**/base.py",
-        """scopes_kwargs = \{"scopes": scopes, "default_scopes": cls\.AUTH_SCOPES\}""",
-        """# Documentai uses a regional host (us-documentai.googleapis.com) as the default
-            # so self-signed JWT cannot be used.
-            # Intentionally pass default scopes as user scopes so the auth library
-            # does not use the self-signed JWT flow.
-            # https://github.com/googleapis/python-documentai/issues/174
-            scopes_kwargs = {"scopes": scopes or cls.AUTH_SCOPES, "default_scopes": cls.AUTH_SCOPES}""")
-
     s.replace(library / "tests/**/documentai_v1beta2/*.py",
```
Comment (busunkim96) [inline owlbot.py:39]: I believe you'll want to also delete the modifications to the unit tests. After that, please re-run owlbot using the commands @parthea shared.

Comment (parthea) [reply owlbot.py]: Done.

**Review by parthea — APPROVED** [review]

**Pull request merged**